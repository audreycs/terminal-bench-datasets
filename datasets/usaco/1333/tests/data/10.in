10
213394634718308650 567314195893526830
118718333481546670 984222666619725683
521169680133050475 379632545405725706
529433210710147101 666030768028277123
248727345489185865 923284915041497532
555027965912713377 803887041840034130
349121089908475057 822348686848065296
607629682756169018 642020332228675401
946903533858245439 44225285843701573
981228638057593 912509743147346586
