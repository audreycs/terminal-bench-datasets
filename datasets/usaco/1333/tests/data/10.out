276
244
207
472
1104
333
1074
288
590
1498
